### DISCUSSION TITLE
Improvement - Shortcut Edit-Button (Button of Death)

### DISCUSSION THREAD

**Discussion by WalkingDot** [AppGround-io/bluetooth-keyboard-and-mouse-support#141] [Ideas]
Hi,
can you move the Edit-Shortcut-Button into a menu, please? Or maybe after the activation of shortcuts, just show a short popup?

Unfortunately, when playing games, the Edit-Button is touched far too often.

![20200709_033819](https://user-images.githubusercontent.com/67340594/86987184-b178e700-c195-11ea-9a8b-d1bc1ed17df5.jpg)

**Comment by AppGround-io** [comment]
In the current beta release the edit button is now at the beginning of the shortcuts and made it smaller. Let me know if this fixes your problem.

![device-2020-07-22-093639](https://user-images.githubusercontent.com/50548429/88148598-1c4b0900-cbff-11ea-98dc-64baf86423f2.png)

**Comment by WalkingDot** [comment]
Now the edit button is closer to the left mouse button and is touched even more often. Previously it was sometimes triggered by the ball of the thumb.

Would something like this be possible?:
![Screenshot_20200723-032405_Photos](https://user-images.githubusercontent.com/67340594/88244766-11db4e80-cc95-11ea-8243-90249969a744.jpg)

**Comment by AppGround-io** [comment]
Adding a button in the menu would be a lot of work, because it's not supported by the current menu provided by the system and I would need to create my menu just to add a button. 

Maybe we find another solution for this problem.

Can you explain a little bit more why this new position is not working? 
How often do you accidentally press the edit button in the new position?
Do you also press other shortcut buttons accidentally?

Maybe more space between the shortcut buttons would sense instead of moving the edit button.

**Comment by WalkingDot** [comment]
Too bad that the menu can't be customized but thank you for trying.

It depends a lot on the game and the shortcuts set for it. RPGs work very well but Hack and Slays like Diablo are more hectic and if there is an unexpected escape or the need to take potions, often in these moments a click on the Edit Button happens.

I touch the Edit Button maybe once every 15 minutes.
I also touch accidentally, at a range of ~2 cm, other shortcuts (e.g. potions or switching a skill) but the mouse still works and it is not disturbing. That's why I suspect that more space won't help much.

**Comment by zorazor** [comment]
Maybe a double tap or a swipe to unlock the button?

**Comment by AppGround-io** [comment]
@zorazor Double tap could work, but now that you mention it, long press is easy to implement, so I will try this for the next version. Thank you for the inspiration, haven't thought about it before.

**Comment by WalkingDot** [comment]
Yay! I think this is gonna be awesome. @zorazor and @AppGround-io thank you so much!

@AppGround-io 
Sorry for asking but please could you put the Edit Button back to the bottom corner?

**Comment by AppGround-io** [comment]
The change is now available in the beta version, I also moved the button back to the old place.

**Comment by WalkingDot** [comment]
I have tried it over a longer time. It is nearly perfect and couch gaming is really fun now.
Could you add one more second for the activation, please? I think than it is perfect 🥇

**Comment by WalkingDot** [comment]
> Adding a button in the menu would be a lot of work, because it's not supported by the current menu provided by the system and I would need to create my menu just to add a button.

Since a while there is the clearly organized new menu.
Could you please add the shortcut edit-button to the menu?
